### PR TITLE
release notes: high level sig-auth release notes for 1.8

### PR DESCRIPTION
**Pull request by ericchiang**
cc @kubernetes/sig-auth-misc

### CODE REVIEW

**Comment by Bradamant3** [comment]
Hope it's OK to take this one to merge; there was a conflict and I didn't want to perpetuate any more.

File: release-1.8/release_notes_draft.md [hunk 1]
```diff
@@ -73,6 +79,9 @@ API surface for promotion.
   any domain name prefix outside of the `kubernetes.io/` domain instead of the
   previous `pod.alpha.kubernetes.io/opaque-int-resource-` prefix.
 
+### Auth
+- With the introduction of RBAC v1, the RBAC v1alpha has been depricated.
```
Comment (destijl) [inline release-1.8/release_notes_draft.md]: deprecated

File: release-1.8/release_notes_draft.md [hunk 2]
```diff
@@ -172,6 +178,13 @@ kind.
   metrics) are graduating to v1beta1.  See [Action Required Before
   Upgrading](#action-required-before-upgrading).
 
+### Auth
+
+* [GA] The RBAC API group has been promoted from v1beta1 to v1. No API changes were introduced.
+* [beta] Advanced auditing has been promoted from alpha to beta. The webhook and logging policy formats have changed since alpha, and may require modification.
+* [beta] Kubelet certificate rotation through the certificates API has been promoted from alpha to beta. RBAC cluster roles for the certificates controller have been added for common uses of the certificates API, such as the kubelet's.
+* [alpha] The API server can now use Google Cloud KMS for encryption of resources, such as secrets, before storing them in etcd.
```
Comment (destijl) [inline release-1.8/release_notes_draft.md]: [alpha] Building on the 1.7 work to allow encryption of resources such as secrets, a mechanism to store keys in external Key Management Systems (KMS) was introduced. This complements the original file-based storage and allows integration with multiple KMS. A Google Cloud KMS plugin was added and will be usable once the Google side of the integration is complete.

@sakshamsharma @jcbsmpsn

**Comment by destijl** [comment]
Argh, sorry, didn't notice this was merged.

**Pull request merged**